tests/filecheck/parser-printer/llvm_mlir_printer.xdsl

```diff
@@ -1,17 +0,0 @@
-// RUN: xdsl-opt -t mlir %s | filecheck %s
```

Comment:
Why was this file removed? Do we not support this functionality anymore?

Reply:
This file was a subset of the `llvm_test.xdsl` one. So now this line is in `llvm_test.xdsl`